catalog/Web_Apps_Services_Interaction/API_Builders.yml

```diff
@@ -17,6 +17,9 @@ projects:
   - grapethor
   - jb
   - jbuilder
+  - jsonapi-hanami
+  - jsonapi-rails
+  - jsonapi-ruby
```

Comment:
@factcondenser This gem doesn't seem to exist, could you please adjust it to the proper name or remove it?